pyproject.toml

```diff
@@ -57,6 +57,13 @@ doc = [
   "bokeh"
 ]
 
+extra = [
```

Comment:
Why has the `extra` target been added? I'd have thought that `arviz` would have been added to `doc`.

I know that @astrofrog  has been working on PRP 771 - e.g. https://discuss.python.org/t/pep-771-default-extras-for-python-software-packages/79706/76 - but I don't think it's been approved yet.

Reply:
It should also be added to docs.

What I'm trying to do here is to have a group for all the optional extras that user might want (that don't want to build the docs). Bokeh could go in there and if I make an interface to e.g. export fit results to pandas, then pandas would go in there.

Many users might just want everything and the kitchen sink as long as they can get it with pip.

Reply:
I'd rather have the discussion of what goes in an `extra` argument in a separate PR, as it isn't related to this one.

Reply:
Will remove on next push. Since most users presumably install for conda anyway, they won't use `pip install "sherpa[extra,test,doc]"` anyway, so they won't benefit for it. If we add more packages, I think we should add such a group, but it doesn't have to be now.